### PR TITLE
WIP Tutorial on the many ways to access IPFS content

### PR DESCRIPTION
**Pull request by flyingzumwalt**
Entry point for the new Tutorial: https://github.com/flyingzumwalt/decentralized-web-primer/blob/avenues-for-access/avenues-for-access/README.md

### CODE REVIEW

File: avenues-for-access/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+* Access content through the public IPFS gateways at ipfs.io
+* Access content through any IPFS node's http gateway
+* Access content using the IPFS browser extension
+* Access IPFS content through Tor
+* Use a sneakernet to move and redistribute IPFS content
+* Explain the implications of being able to access IPFS content through so many different paths
+
+## Lessons
+
+1. [The Power of Content-addressing](/avenues-for-access/lessons/power-of-content-addressing.md)
```
Comment (whyrusleeping) [inline avenues-for-access/README.md]: number is weird here, on purpose?

File: avenues-for-access/lessons/browser-extension.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Lesson: Access IPFS content through a browser extension
+
+This is a placeholder. There are currently four web browser extensions that help your retrieve content from IPFS. Each works in slightly different ways. We are in the process of consolidating that code and making it more secure before we encourage people to rely on it. When that work is complete, it will be possible to interact directly with the IPFS network by simply installing a browser extension -- no HTTP gateway needed, nothing else to install on your computer, no need to use the command line.
+
+We consider this the next big step to getting IPFS natively supported in web browsers.  You can track this work in the github repository at https://github.com/ipfs/in-web-browsers. [This comment on a github issue](https://github.com/ipfs/pm/issues/351#issuecomment-294262546) describes the state of these efforts as of April 2017.
```
Comment (whyrusleeping) [inline avenues-for-access/lessons/browser-extension.md]: Maybe describe the approach to how an extension would work in the browser?

File: avenues-for-access/lessons/local-node.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+If the daemon is not running, your IPFS node won't be able to retrieve content from other nodes on the network. It also won't start the HTTP gateway that you're going to use in Step 3.
+
+### Step 2: Read the content on the command line
+
+For example, if you would like to use the command-line to explore the snapshot of wikipedia that we're using as an example in all of the lessons in the [Tutorial on Avenues for Access](/avenues-for-access/), you can do so with commands like this:
+
+```
+# get the article for Content addressable networks
+ipfs cat QmQP99yW82xNKPxXLroxj1rMYMGF6Grwjj2o4svsdmGh7S > Content_addressable_network.html
```
Comment (whyrusleeping) [inline avenues-for-access/lessons/local-node.md]: this is the hash for the wikipedia dump, are you sure you copy pasted corectly?

File: avenues-for-access/lessons/other-gateways.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+To view the wikipedia page we're using as an example in all of the lessons in the [Tutorial on Avenues for Access](/avenues-for-access/), use these links:
+
+- 2017-04-30 snapshot: http://dweb.link/ipfs/QmQP99yW82xNKPxXLroxj1rMYMGF6Grwjj2o4svsdmGh7S
+- latest (IPNS): http://dweb.link/ipns/QmQP99yW82xNKPxXLroxj1rMYMGF6Grwjj2o4svsdmGh7S
+- latest (DNS): http://dweb.link/ipns/wikipedia-on-ipfs.io
+
+## Explanation
+
+TODO
+Restricting the content that your gateway will serve
```
Comment (whyrusleeping) [inline avenues-for-access/lessons/other-gateways.md]: unfortunately right now, this needs to be done with an nginx config. There arent any built in things for restricting what the gateway serves yet

File: avenues-for-access/README.md (hunk 1 above)
Comment (flyingzumwalt) [inline avenues-for-access/README.md]: fixed

File: avenues-for-access/lessons/local-node.md (hunk 3 above)
Comment (flyingzumwalt) [inline avenues-for-access/lessons/local-node.md]: fixed

File: avenues-for-access/lessons/other-gateways.md (hunk 4 above)
Comment (flyingzumwalt) [inline avenues-for-access/lessons/other-gateways.md]: leaving this un-written for now.

**Pull request merged**